driverkit/utils/generate

```diff
@@ -46,16 +44,11 @@ if [ -z ${TARGET_VERSION} ]; then
     exit 1
 fi
 
-ensure_kernelversion() {
-    TARGET_KERNEL_VERSION="$(expr match "${TARGET_KERNEL}" '^.*_\(\d*\)$')"
-
-    # If kernel version is missing
-    test -z $TARGET_KERNEL_VERSION &&\
-	    TARGET_KERNEL_VERSION=$DEFAULT_KERNEL_VERSION &&\
-	    TARGET_KERNEL="${TARGET_KERNEL}_${TARGET_KERNEL_VERSION}"
+compute_kernelversion() {
+    TARGET_KERNEL_VERSION=$(echo "${TARGET_KERNEL##*_}")
```

Comment:
Assume `TARGET_KERNEL` has always the `_$kernelversion` suffix.
This holds true for kernel-crawler; and if it weren't, then it would be a bug on the crawler side.